src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
+
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    arch = attr.ib(type=str, default=None, validator=optional_str)
+    """Architecture type of the image"""
+
+    virt_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Virtualization type """
```

Comment:
What is the difference between this and the already existing attribute `virtualization`?

What are the expected values?

Reply:
virtualization might be the same input. As far as AWS is concerned these are the virtualization types:

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html